Add notes about rendering ads on Windows a11y laptops (#7541)
bbc/simorgh: docs/Rendering-Ads.md

Given you do not have any ad blocking extensions in your browser.

1. Use a proxy to set your location outside the UK. Visiting https://www.test.bbc.com/userinfo should show `"X-Ip_is_advertise_combined": "yes"`. And when you visit https://www.test.bbc.com/mundo you should be able to see an advert.

2. If you would like to see the ads with the debugger tool you can append this query string to the url: `?ads-debug=true`

   i.e https://www.test.bbc.com/mundo?ads-debug=true

## Rendering Ads on an a11y device

> If you are unable to set a proxy on one of the Windows a11y laptops, you won't be able to see ads within the UK. For this reason, you'll have to run Simorgh locally in your working machine and access to it from the Windows laptop following these steps:

Reviewer: ```suggestion
If you are unable to set a proxy on one of the Windows a11y laptops, you won't be able to see ads within the UK. For this reason, you'll have to run Simorgh locally on your working machine (Mac) and allow the Windows laptop to connect to it over LAN with the following steps:
```

1. Connect both devices to the same network

> 2. Update the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` url in the [local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3) file with the IP address of your machine

Reviewer: ```suggestion
2. Update the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` url in the [local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3) file with the IP address of your machine (Mac)
```

Author: I'd prefer not to add Mac, since there could be people using Window machines.

>    i.e `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://192.168.0.50:7080`

Reviewer: ```suggestion
   e.g `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://192.168.0.50:7080`
```

3. Disable the `upgrade-insecure-requests` directive from the [cspHeader](https://github.com/bbc/simorgh/blob/latest/src/server/utilities/cspHeader/index.js#L419)

4. If you can't set the `BBC-Adverts` header in the Windows laptop, you will need to remove [this](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/Canonical/index.jsx#L63) condition from the codebase to be able to render the ad

5. Run Simorgh with a production build `npm run build && npm run start`

> 6. Access to your machine from the Windows laptop

Reviewer: ```suggestion
6. Connect to your machine running Simorgh using the Windows laptop by visiting the IP address of it appended with the correct query strings.
```

>    i.e `http://192.168.0.50:7080/mundo?site=test&ads-debug=true`

Reviewer: ```suggestion
   e.g `http://192.168.0.50:7080/mundo?site=test&ads-debug=true`
```
